daisy_workflows/image_import/import_image.sh

```diff
@@ -48,12 +51,37 @@ function resizeDisk() {
   fi
 }
 
-# Mount GCS bucket containing the disk image.
-mkdir -p /gcs/${SOURCEBUCKET}
-gcsfuse --implicit-dirs ${SOURCEBUCKET} /gcs/${SOURCEBUCKET}
+function copyImageToStagingDisk() {
+  local souceSizeGigabytes="$((1 + SOURCESIZE_BYTES / 1024 / 1024 / 1024))"
+  # We allocate double capacity here since the source file
+  # could be an OVA, which would require making an additional
+  # copy of its enclosed VMDK.
+  local stagingDiskSizeGigabytes="$((souceSizeGigabytes * 2))"
 
-# Atrocious OVA hack.
-SOURCEFILE_TYPE="${$SOURCE_DISK_FILE##*.}"
+  # This disk is initially created with 10GB of space.
+  # Enlarge it if that's insufficient to hold the input image.
+  if [[ ${stagingDiskSizeGigabytes} -gt 10 ]]; then
+    resizeDisk "${STAGING_DISKNAME}" "${stagingDiskSizeGigabytes}" "${ZONE}"
+  fi
+
+  local stagingPath="/gcs/$(dirname ${SOURCEPATH})/"
```

Comment:
/gcs/ is not an appropriate directory name anymore as it's not linked to gcs. It can lead someone to assume it's linked to GCS bucket as before when it's not.

Reply:
ahh, that's a good point. switched to `/daisy-scratch/`